src/components/organisms/transaction/TransactionItem.js

```diff
@@ -0,0 +1,149 @@
+
+const styles = StyleSheet.create({
+    transactionPreview: {
+        width: '100%',
+        borderRadius: 6,
+        backgroundColor: GlobalStyles.color.WHITE,
+    },
+    date: {
+        fontSize: 10,
+    },
+    iconContainer: {
+        marginRight: 14,
+    },
+    amounteOutgoing: {
+        color: GlobalStyles.color.RED,
+    },
+    amountIncoming: {
+        color: GlobalStyles.color.GREEN,
+    },
+    address: {
+        fontSize: 11,
+    },
+    bold: {
+        fontSize: 11,
+    },
+});
+
+type Props = {
+    transaction: TransactionModel,
+};
+
+export default class TransactionItem extends Component<Props, State> {
+    iconName = () => {
+        return this.props.transaction.type;
+    };
+
+    title = () => {
+        return translate('transactionTypes.' + this.props.transaction.type);
+    };
+
+    renderAction = () => {
+        return;
+    };
+
+    renderDetails = () => {
+        return;
+    };
+
+    renderAddress = () => {
+        const { transaction } = this.props;
+        return <Trunc type="address">{transaction.signerAddress || '-'}</Trunc>;
+    };
+
+    render = () => {
+        const { transaction, showDetails } = this.props;
+        let date = transaction.deadline;
+
+        return (
+            <View style={styles.transactionPreview}>
+                <Row justify="start" fullWidth>
+                    <Col
+                        justify="center"
+                        align="center"
+                        style={styles.iconContainer}
+                    >
+                        <TransactionIcon
+                            size="small"
+                            name={this.iconName()}
+                            style={styles.icon}
+                        />
+                    </Col>
+                    <Col grow>
+                        <Row justify="space-between">
+                            <Text type="regular" theme="light">
+                                {this.title()}
+                            </Text>
+                            <Text
+                                type="regular"
+                                theme="light"
+                                style={styles.date}
+                            >
+                                {transaction.status === 'unconfirmed'
+                                    ? 'Unconfirmed'
+                                    : date}
+                            </Text>
+                        </Row>
+                        <Row justify="space-between" align="center">
+                            <Text
+                                type="bold"
+                                theme="light"
+                                style={styles.address}
+                            >
+                                {this.renderAddress()}
+                            </Text>
+                            <Row style={styles.value} align="center">
+                                {this.renderAction()}
```

Comment:
does it render anything, it just return; ?

Reply:
> it just return; 
This is stub method. Cichild classes implement these methods.

Removed this component. Will rework it in the next PR.